json-server.py

```diff
@@ -23,6 +39,51 @@ def do_POST(self):
                 status.HTTP_200_SUCCESS.value,
             )
 
+        if url["requested_resource"] == "comments":
+            successfully_created = create_comment(request_body)
+            if successfully_created:
+                return self.response(
+                    "Successfully created", status.HTTP_201_SUCCESS_CREATED.value
+                )
+
+            return self.response(
+                "Invalid data", status.HTTP_400_CLIENT_ERROR_BAD_REQUEST_DATA.value
+            )
+
+        if url["requested_resource"] == "categories":
+            successfully_created = post_category(request_body)
```

Comment:
Same test results as for create_comment().
Works when expected body is provided. Triggers KeyError and no response in postman if missing keys